meta-mender-core/classes/mender-artifactcapsule.bbclass

```diff
@@ -0,0 +1,151 @@
+# ------------------------------ CONFIGURATION ---------------------------------
+
+# Extra arguments that should be passed to mender-artifact.
+MENDER_ARTIFACT_CAPSULE_EXTRA_ARGS ?= ""
+
+# The guid used to sign the mender update.
+MENDER_ARTIFACT_SIGNING_KEY ?= ""
+
+do_image_mender_capsule[depends] += "mender-artifact-native:do_populate_sysroot"
+
+MENDER_ARTIFACT_CAPSULE_NAME ??= "${MENDER_ARTIFACT_CAPSULE_NAME_DEFAULT}"
+MENDER_ARTIFACT_CAPSULE_NAME_DEFAULT = "uefi-capsule-${MACHINE}"
+
+MENDER_ARTIFACT_CAPSULE_NAME_DEPENDS ?= ""
+
+MENDER_ARTIFACT_CAPSULE_PROVIDES ?= ""
+
+MENDER_ARTIFACT_CAPSULE_DEPENDS ?= ""
+
+MENDER_ARTIFACT_CAPSULE_CLEAR_PROVIDES ?= ""
```

Comment:
Nitpick: Should be `CLEARS_PROVIDES` (extra 'S'). This is what the mender-artifact tool uses, and is also what you used below.